modArray.bas

```diff
@@ -350,6 +361,7 @@ Public Character As CharacterData
 Attribute Character.VB_VarUserMemId = 1073741840
 Public Macro(0 To 9) As MacroData
 Attribute Macro.VB_VarUserMemId = 1073741841
+Public Bug(1 To 500) As BugReportData

```

Comment:
What happens at 500?

Reply:
I’m aware of this but haven’t added in the error handling code yet